cookiecutter.json

```diff
@@ -1,16 +1,16 @@
 {
   "_extensions": ["jinja2_time.TimeExtension"],
 
-  "department_name": "Your HM Government department",
+  "organisation_name": "Your public sector organisation name, e.g. GDS",
```

Comment:
Nitpicking here, but encourage full names. Not everyone knows that, for instance, OS is the Ordnance Survey (but also the Office for Students, to add to the confusion)

Reply:
Good spot, my bad! Added in commit [`16902ea`](https://github.com/ukgovdatascience/govcookiecutter/pull/31/commits/16902ea9fed72343297bc6667dce223114facc7a).